src/main/kotlin/org/sert2521/infiniterecharge2020/Main.kt

```diff
@@ -1,23 +1,38 @@
 package org.sert2521.infiniterecharge2020
 
+import kotlinx.coroutines.launch
+import org.sert2521.infiniterecharge2020.autonomous.PathGenerator
+import org.sert2521.infiniterecharge2020.autonomous.centerInitPowerPort
+import org.sert2521.infiniterecharge2020.climber.Climber
 import org.sert2521.infiniterecharge2020.drivetrain.Drivetrain
 import org.sert2521.infiniterecharge2020.powerhouse.PowerHouse
 import org.sert2521.infiniterecharge2020.powerhouse.closeHouse
 import org.sert2521.sertain.events.onEnable
-import org.sert2521.sertain.events.whileTeleop
+import org.sert2521.sertain.events.whileAuto
 import org.sert2521.sertain.robot
+import org.sert2521.sertain.subsystems.access
 import org.sert2521.sertain.subsystems.add
 
 suspend fun main() = robot {
     println("Robot program starting")
     add<Drivetrain>()
+    add<Climber>()
     add<PowerHouse>()
+    add<PathGenerator>()
 
     onEnable {
         closeHouse()
     }
 
-    whileTeleop {
+    launch {
```

Comment:
Why launch not whileTeleop

Reply:
🤔🤔🤔🤔 Yeah it should be whileTeleop. Good catch!